dts/bindings/interrupt-controller/wch,pfic.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Michael Hope
+# SPDX-License-Identifier: Apache-2.0
+
+description: RV32M1 INTMUX IP
```

Comment:
This not seems to be right.

Reply:
CC @nzmichaelh for help with this one (`description: RV32M1 INTMUX IP` problem).

Reply:
My fault. I used that as a template. Use "WCH CH32V00x Programmable Fast Interrupt Controller (PFIC)" instead.

optional: the descriptions across the new bindings are inconsistent. Consider using "WCH CH32V00x $FULL_NAME ($ACRONYM)" for all.

Reply:
Thanks! I have now updated the descriptions across all of the new bindings.